Index: src/main/webapp/resources/js/topjava.common.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/webapp/resources/js/topjava.common.js	(date 1543424377000)
+++ src/main/webapp/resources/js/topjava.common.js	(date 1543424701229)
@@ -12,10 +12,9 @@
 function deleteRow(id) {
     $.ajax({
         url: ajaxUrl + id,
-        type: "DELETE",
-        success: function () {
-            updateTable();
-        }
+        type: "DELETE"
+    }).done(function () {
+        updateTable();
     });
 }
 
@@ -26,14 +25,13 @@
 }
 
 function save() {
-    var form = $("#detailsForm");
+    let form = $("#detailsForm");
     $.ajax({
         type: "POST",
         url: ajaxUrl,
-        data: form.serialize(),
-        success: function () {
-            $("#editRow").modal("hide");
-            updateTable();
-        }
+        data: form.serialize()
+    }).done(function () {
+        $("#editRow").modal("hide");
+        updateTable();
     });
 }
\ No newline at end of file
Index: src/main/webapp/resources/js/topjava.users.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/webapp/resources/js/topjava.users.js	(date 1543424377000)
+++ src/main/webapp/resources/js/topjava.users.js	(date 1543424532094)
@@ -1,5 +1,5 @@
-var ajaxUrl = "ajax/admin/users/";
-var datatableApi;
+const ajaxUrl = "ajax/admin/users/";
+let datatableApi;
 
 // $(document).ready(function () {
 $(function () {
